client/my-sites/draft/index.jsx

```diff
@@ -196,6 +196,11 @@ module.exports = React.createClass( {
 						{ post.format === 'aside' ? excerpt : title }
 					</a>
 				</h3>
+				{ post.excerpt &&
```

Comment:
The display of excerpt looks a little odd on existing posts screen when excerpt is long (clips):

![image](https://cloud.githubusercontent.com/assets/1779930/23367106/129941f6-fcd7-11e6-89a1-dfdcad2b81f1.png)

Trying to figure out the special casing for `aside` types (see condition a few lines up using `excerpt`). Would this show the excerpt double now?